File: insurance-internal-app/src/main/webapp/internal-view/src/app/insurance-policy/insurance-policy-checkout-request.ts
import {InsurancePolicyRequest} from './insurance-policy-request';
import {InsurancePolicyCarRequest} from './insurance-policy-car-form/insurance-policy-car-request';
import {InsurancePolicyHomeRequest} from './insurance-policy-home-form/insurance-policy-home-request';

export class InsurancePolicyCheckoutRequest {

    insurancePolicyRequest:  InsurancePolicyRequest;
    insurancePolicyCarRequest: InsurancePolicyCarRequest;
    insurancePolicyHomeRequest: InsurancePolicyHomeRequest; 

    constructor(insurancePolicyRequest: InsurancePolicyRequest,
        insurancePolicyCarRequest: InsurancePolicyCarRequest,
        insurancePolicyHomeRequest: InsurancePolicyHomeRequest) {
        this.insurancePolicyRequest = insurancePolicyRequest;
        this.insurancePolicyCarRequest = insurancePolicyCarRequest;
        this.insurancePolicyHomeRequest = insurancePolicyHomeRequest;
    } 

}

File: model/src/main/java/model/request/InsurancePolicyCheckoutRequest.java
package model.request;

public class InsurancePolicyCheckoutRequest {

	private InsurancePolicyRequest insurancePolicyRequest;
	private InsurancePolicyCarRequest insurancePolicyCarRequest;
	private InsurancePolicyHomeRequest insurancePolicyHomeRequest;

	public InsurancePolicyCheckoutRequest() {
		super();
	}

	public InsurancePolicyCheckoutRequest(InsurancePolicyRequest insurancePolicyRequest,
			InsurancePolicyCarRequest insurancePolicyCarRequest,
			InsurancePolicyHomeRequest insurancePolicyHomeRequest) {
		super();
		this.insurancePolicyRequest = insurancePolicyRequest;
		this.insurancePolicyCarRequest = insurancePolicyCarRequest;
		this.insurancePolicyHomeRequest = insurancePolicyHomeRequest;
	}

	public InsurancePolicyRequest getInsurancePolicyRequest() {
		return insurancePolicyRequest;
	}

	public void setInsurancePolicyRequest(InsurancePolicyRequest insurancePolicyRequest) {
		this.insurancePolicyRequest = insurancePolicyRequest;
	}

	public InsurancePolicyCarRequest getInsurancePolicyCarRequest() {
		return insurancePolicyCarRequest;
	}

	public void setInsurancePolicyCarRequest(InsurancePolicyCarRequest insurancePolicyCarRequest) {
		this.insurancePolicyCarRequest = insurancePolicyCarRequest;
	}

	public InsurancePolicyHomeRequest getInsurancePolicyHomeRequest() {
		return insurancePolicyHomeRequest;
	}

	public void setInsurancePolicyHomeRequest(InsurancePolicyHomeRequest insurancePolicyHomeRequest) {
		this.insurancePolicyHomeRequest = insurancePolicyHomeRequest;
	}

}
